R/bambu-extendAnnotations-utilityExtend.R

```diff
@@ -44,17 +44,17 @@ isore.extendAnnotations <- function(combinedTranscripts, annotationGrangesList,
   }
 }
 
-#' filter transcripts by read counts and NDR
+#' filter transcripts by read counts
 #' @noRd
-filterTranscripts <- function(combinedTranscripts, min.sampleNumber, max.txNDR){
+filterTranscripts <- function(combinedTranscripts, min.sampleNumber){
+  filterSet = NULL
```

Comment:
when filterSet is NULL, I think sometimes, `combinedTranscripts[filterSet,]` will throw error as there is nothing to select. Suggest to remove 
```suggestion
```